python/cuml/test/dask/conftest.py

```diff
@@ -10,6 +12,9 @@
 
 @pytest.fixture(scope="module")
 def cluster():
+
+    dask.config.set({"distributed.comm.timeouts.connect": "50s"})
```

Comment:
Curiosity question, why 50 seconds in particular (as opposed to a minute for example?). Not that it matters, just wondering how did you choose

Reply:
I had copied this from an example somewhere and just left it at 50s. For some reason 10s was not enough time and I think that might be from the limited resources in CI as well as the non-determinism in garbage collection across the tests. I think it's likely 25 seconds would have been enough, but I raised it a little further just to avoid having to increase it again in the near future as we continue to build tests (and limited CI resources continue to cause pain). 

That's just the long way of saying, I can increase it to 60s if you'd like.